### PR TITLE
check mean time to merged

### PR DESCRIPTION
**Pull request by daohien**
create PR

### CODE REVIEW

**Comment by daohien** [comment]
Lỗi config, update

**Comment by daohien** [comment]
Merge

**Comment by duclh-3136** [comment]
comment 1

**Comment by duclh-3136** [comment]
them cmt nua

**Comment by duclh-3136** [comment]
them nua ne

File: project.html
[the remark this answers is not in the document]
Comment (duclh-3136) [reply project.html:1]: review +8

**Pull request merged**